shared/common/common.go

```diff
@@ -39,3 +41,22 @@ func CreateDirWithFileMode(dirPath string, perm os.FileMode) error {
 	}
 	return nil
 }
+
+// GetRootDir gets directory of git root
+func GetRootDir() (string, error) {
+	cmd := exec.Command("git", "rev-parse", "--show-toplevel")
+	output, err := cmd.Output()
+	if err != nil {
+		return "", err
+	}
+	return strings.TrimSpace(string(output)), nil
+}
+
+// CDToRootDir change directory to git root dir
+func CDToRootDir() error {
```

Comment:
changeToRootDir or MoveToRootDir?

Reply:
I know CD is not a good verb in a function, but among the others, CD feels more natural to me to be honest